Android.bp

```diff
@@ -33,7 +33,7 @@ mfx_cc_defaults {
     export_include_dirs: ["include"],
 
     include_dirs: [
-        "hardware/intel/external/minigbm-intel/cros_gralloc", // remove then minigbm gets Android.bp
```

Comment:
CiV will use hardware/intel/external/minigbm-intel/cros_gralloc, but not external/minigbm